assets/tests.py

```diff
@@ -0,0 +1,145 @@
+import json
+from automationcommon.tests.utils import UnitTestCase
+from rest_framework.test import APIClient
+
+
+class APIViewsTests(UnitTestCase):
+    def setUp(self):
+        self.maxDiff = None
+
+    def assertDictListEqual(self, d1, d2, msg=None):
+        """Compares two dictionary with lists (order doesn't matter)"""
```

Comment:
Should one perhaps ``copy.copy`` the dictionaries first? Otherwise this method silently modifies d1 and d2, yes?

Reply:
makes sense, if the test continues, then it will break the dictionary